ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -390,6 +390,14 @@ tests:
     test:
     - ref: openstack-noop
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-azurestack
+  steps:
+    cluster_profile: azure4
```

Comment:
I think we're going to need a new `cluster_profile` This recent PR for arm support should give an idea of how to do it: https://github.com/openshift/ci-tools/pull/2097

Reply:
Isn't AzureStack supposed to be another variant of Azure similar to GovCloud? I only ask this because the UPI script uses Azure subscription present in the azure4 cluster profile to login and connect to the ppe2 environment. Would this profile solve the cluster limiting?

Reply:
> Isn't AzureStack supposed to be another variant of Azure similar to GovCloud? 
This is correct. GovCloud should probably have its own cluster profile too. Both ASH & GovCloud interact with different environments than public Azure. 

> I only ask this because the UPI script uses Azure subscription id present in the azure4 cluster profile to login and connect to the ppe2 environment. Would this profile solve the cluster limiting?

These steps are no longer needed in UPI. Prior to openshift/installer#5024 the installer would throw an error if trying to connect to an ASH environment. As a workaround, we would connect and create the manifests for public Azure, then modify them through scripts to work for ASH. Now that openshift/installer#5024 has merged, the installer should create the manifests correctly without needing to connect to Azure.